README.md

```diff
@@ -1,4 +1,4 @@
-# Awesome Deep Learning [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
```

Comment:
what is this change?

Reply:
I've added GAN and Fast R-CNN research papers in the research section.

Reply:
correct. that change should be for lines 157
But why is this changed?